easybuild/easyconfigs/x/x265/x265-3.6-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'CMakeMake'
+
+name = 'x265'
+version = '3.6'
+
+homepage = 'https://x265.org/'
+description = """
+ x265 is a free software library and application for encoding video streams
+ into the H.265 AVC compression format, and is released under the terms of
+ the GNU GPL.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+source_urls = ['https://bitbucket.org/multicoreware/x265_git/downloads/']
+sources = ['%(name)s_%(version)s.tar.gz']
+checksums = ['663531f341c5389f460d730e62e10a4fcca3428ca2ca109693867bc5fe2e2807']
+
+builddependencies = [
+    ('binutils', '2.42'),
+    ('CMake', '3.29.3'),
+    ('Yasm', '1.3.0'),
+]
+
+configopts = '-DGIT_ARCHETYPE=1'
```

Comment:
you removed `-DENABLE_PIC=ON` here, is that intentional?

Reply:
Oh, that was unintentional. Fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/21241/commits/4ee2a2d3c83587cbfe1c4914d465cd5c5ea72a4a